aurora-mysql-plugin/src/test/java/io/cdap/plugin/aurora/mysql/AuroraMysqlPluginTestBase.java

```diff
@@ -115,9 +109,19 @@ public static void setupTest() throws Exception {
       BASE_PROPS.get(ConnectionConfig.PORT) + "/" + BASE_PROPS.get(ConnectionConfig.DATABASE);
     Connection conn = createConnection();
     createTestTables(conn);
+    tearDown = true;
```

Comment:
I assume we're doing this because the tear down can fail if the expected tables don't exist?

This is not a good way to guard against that type of scenario though, as it's possible to successfully create some tables but fail partway through. Then tearDown never gets set and the tables don't get cleaned up.

Instead, it's better not to have this flag at all. In the cleanup section, if we can't drop a table, we log a message and move on.

Reply:
Thanks for the suggestion. Done :)